var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -105,6 +109,8 @@ class Qt(Package):
     # depends_on("ogg", when='+multimedia')
 
     use_xcode = True
+    # parallel build fails on Sierra
+    parallel = False
```

Comment:
Ouch. This package already takes so long to build. We could turn this into a property that returns True for every OS other than darwin. But I'm surprised it only fails on darwin. Have you tried contacting the developers about this? I've found a lot of developers very open to fixing Makefiles to work in parallel. Also, is it the build phase that fails in parallel or just install? Because you can override a single phase to run in serial if that's all you need.